package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.22.0",
   "description": "Get a logger. Any logger.",
   "main": "anylogger.js",
+  "typings": "anylogger.d.ts",
```

Comment:
Canonical name for this property is `types`, see https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html.

Reply:
Cheers for that.